src/helm/proxy/clients/auto_client.py

```diff
@@ -68,6 +68,9 @@ def _get_client(self, model: str) -> Client:
 
             if get_huggingface_model_config(model):
                 client = HuggingFaceClient(cache_config=cache_config)
+            elif organization == "local":
+                # HACK since we want to use the huggingface sqlite file. TODO avoid this
+                client = HuggingFaceClient(cache_config=cache_config)
```

Comment:
I don't think we need this if we just make the group `huggingface` rather than `local`. Likewise below.